apps/playground/src/app/image/page.tsx

```
@@ -0,0 +1,128 @@
+				"GET",
+				"/orgs/{id}/projects",
+				{
+					params: {
+						path: {
+							id: orgId,
+						},
+					},
+				},
+			)) as { projects: Project[] };
+		} catch (error) {
+			console.warn("Failed to fetch projects for organization:", orgId, error);
```

ESLint

[error] Unexpected console statement.

(no-console)